datahub-web-react/src/app/identity/user/ViewInviteTokenModal.tsx

```diff
@@ -79,7 +82,15 @@ export default function ViewInviteTokenModal({ visible, onClose }: Props) {
                 <ModalSectionParagraph>
                     Generate a new invite link! Note, any old links will <b>cease to be active</b>.
                 </ModalSectionParagraph>
-                <CreateInviteTokenButton onClick={() => createNativeUserInviteToken({})} size="small" type="text">
+                <CreateInviteTokenButton
+                    onClick={() => {
+                        if (canManageUserCredentials) {
```

Comment:
Do we need this check?

Reply:
I don't think so but I also feel like it doesn't hurt?

Reply:
Removed